manifold_database_dump_dag.py

```diff
@@ -0,0 +1,94 @@
+"""Airflow DAG to Run the Blogs Sync rake task on a Manifold instance"""
+from datetime import datetime, timedelta
+import airflow
+from airflow.operators.bash_operator import BashOperator
+from airflow.hooks.base_hook import BaseHook
+from airflow.models import Variable
+from airflow.contrib.operators.ssh_operator import SSHOperator
+from airflow.contrib.operators.sftp_operator import SFTPOperator
+from airflow.contrib.operators.sftp_to_s3_operator import SFTPToS3Operator
+# from airflow.operators.python_operator import PythonOperator
+from manifold_airflow_dags.task_slack_posts import slackpostonfail
+# from manifold_airflow_dags.task_slack_posts import slackpostonsuccess
+
+MANIFOLD_INSTANCE_SSH_CONN = \
+        BaseHook.get_connection("AIRFLOW_CONN_MANIFOLD_SSH_INSTANCE")
+MANIFOLD_DUMP_DATABASE_INTERVAL = \
+        Variable.get("MANIFOLD_DUMP_DATABASE_SCHEDULE_INTERVAL")
+DUMP_PATH = "/tmp"
+DUMP_FILENAME = f"manifold-{ datetime.now().strftime('%Y-%m-%dT%H%z')}.sqlc"
+DEFAULT_ARGS = {
+    'owner': 'airflow',
+    'start_date': datetime(2019, 5, 28),
+    'email': ['steven.ng@temple.edu'],
+    'email_on_failure': False,
+    'email_on_retry': False,
+    'retries': 0,
+    'on_failure_callback': slackpostonfail,
+    'retry_delay': timedelta(minutes=5),
+}
+
+
+DAG = airflow.DAG(
+    'manifold_database_dump',
+    default_args=DEFAULT_ARGS,
+    catchup=False,
+    max_active_runs=1,
+    schedule_interval=MANIFOLD_DUMP_DATABASE_INTERVAL,
+)
+
+#
+# CREATE TASKS
+#
+# Tasks with all logic contained in a single operator can be declared here.
+# Tasks with custom logic are relegated to individual Python files.
+
+# Note: single quotes are required in the bash command line.
+dump_database_bash = f"""
+sudo su - postgres bash -c \
+  'pg_dump --format=c --file={ DUMP_PATH }/{ DUMP_FILENAME } manifold'
+"""
+
+dump_database = SSHOperator(
+    task_id='dump_database',
+    command=dump_database_bash,
+    ssh_conn_id="manifold-db",
+    dag=DAG
+)
+
+sftp_get_db_dump = SFTPOperator(
+    task_id='sftp_get_db_dump',
+    ssh_conn_id="manifold-db",
+    remote_filepath=f"{ DUMP_PATH }/{ DUMP_FILENAME }",
+    local_filepath=f"/usr/local/airflow/{ DUMP_FILENAME }",
+    operation="get",
+    dag=DAG
+)
+
+#
+# Data Bucket Variables
+#
+
+AIRFLOW_S3 = BaseHook.get_connection("AIRFLOW_S3")
+AIRFLOW_DATA_BUCKET = Variable.get("AIRFLOW_DATA_BUCKET")
+sftp_put_db_dump_to_s3 = SFTPToS3Operator(
+    task_id='sftp_put_db_dump_to_s3',
+    s3_key=AIRFLOW_S3.login,
```

Comment:
Key here is not the s3 login information. Key is the term s3 uses for it's files. This should be the filename, including directory paths, or "prefixes" as s3 refers to them. 

Shoudl probably be something like f`manifold-db-backups/{DUMP_FILENAME}`